vagrant-spk

```diff
@@ -113,7 +117,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provider :libvirt do |libvirt, override|
     libvirt.cpus = cpus
     libvirt.memory = assign_ram_mb
-    libvirt.random_hostname = true
+    libvirt.default_prefix = PROJECT_BASENAME + "_sandstorm_#{Time.now.utc.to_i}"
```

Comment:
+0.9, I prefer randomness but then again there's nothing wrong with this in practice for this use.